tests/manage/z_cluster/upgrade/test_check_osd_post_ocs_upgrade.py

```diff
@@ -0,0 +1,32 @@
+import logging
```

Comment:
We have test "test_check_mon_pdb_post_upgrade" in the same upgrade directory.
I feel we can combine both mon and osd pdb check tests as both tests needs same verification

Reply:
Moved